### ISSUE TITLE
Analysis code

### ISSUE THREAD

**Issue by Yukalabo** [OpenMendel/OrdinalGWAS.jl#39]
I appreciate this wonderful resource.
I have questions about specifying snpmodel and output analysis results.

**1. specifying snpmodel**
I wanted to analyze with a dominant model, so I entered 
[using OrdinalGWAS
ordinalgwas(@formula(phenotype ~ covariate), phenotype-data , genotype-data; geneticformat = "VCF", vcftype = :GT, snpmodel = "DOMINANT_MODEL", pvalfile="pval.txt.gz")"]
and got the following error message;
[TypeError: in keyword argument snpmodel, expected Union{Val{1}, Val{2}, Val{3}}, got a value of type String]. 

Val(1), Val(2), and Val(3) mean additive, dominant, and recessive model, respectively?

**2.  details of the analysis results**
What code should I enter to output the odds ratio and the 95% Confidence interval?

It is my first time to using Julia, sorry for the basic questions. I look forward to your reply.

**Comment by kose-y** [comment]
Yes, and you need to put just `DOMINANT_MODEL`; it's the name of the predefined symbol, not a string. It automatically translates to one of `Val{1}`, `Val{2}`, and `Val{3}`.

**Comment by kose-y** [comment]
I don't think we have a way to print out the confidence interval or odds ratio now. @chris-german and @Hua-Zhou might have a better answer to this.

**Comment by kose-y** [comment]
The name is in the `SnpArrays` packages, so you will need to import it (e.g., `using SnpArrays`) before using them.
